servers/rendering/renderer_rd/storage_rd/render_scene_buffers_rd.cpp

```diff
@@ -179,8 +179,8 @@ void RenderSceneBuffersRD::configure(const RenderSceneBuffersConfiguration *p_co
 		texture_samples = RD::TEXTURE_SAMPLES_1;
 	} else {
 		texture_samples = msaa_to_samples(msaa_3d);
-		create_texture(RB_SCOPE_BUFFERS, RB_TEX_COLOR_MSAA, base_data_format, get_color_usage_bits(false, true, can_be_storage), texture_samples);
```

Comment:
Been meaning to test it but haven't had time, but I believe we should mark `RB_TEX_COLOR` and `RB_TEX_DEPTH` as transient (or whichever name we decide on) as well.

In the most optimal scenario (no post effects like glow/dof and not reading from screen/depth texture), depth never needs to be written to texture memory. Same with color which will be resolved into our render target texture in the 2nd subpass and thus also doesn't need to be written out to texture memory either. Both should have a measurable impact on Quest.